target/linux/realtek/image/Makefile

```diff
@@ -21,6 +21,11 @@ define Build/dlink-cameo
 	$(SCRIPT_DIR)/cameo-tag.py $@ $(DLINK_KERNEL_PART_SIZE)
 endef
 
+define Build/dlink-version
+	echo -n "OpenWrt" >> $@
+	dd if=/dev/zero bs=1 count=25 >> $@
```

Comment:
I usually do this the other way around `bs=25 count=1` for higher throughput. But I guess we're talking about minor performance gains again here :)